buildspec/linuxIntegrationTests.yml

```diff
@@ -18,7 +18,7 @@ phases:
             # Install .NET Core 2.1 and VSCode
             - apt-get install -y dotnet-sdk-2.1 code
             # Install other needed dependencies
-            - apt-get install -y python2.7 python3.7 docker
+            - apt-get install -y python2.7 python3.8 docker
```

Comment:
Does Python 3.7 come with 3.8 or are we completely dumping 3.7?

Reply:
I'll look into the integration tests separately since they are having broader issues. I'm unsure why python 3.6 isn't in there either.

Reply:
We need all of them, maybe we can leverage the pyenv work